Add Initiatives to Admin manual (#36)
decidim/documentation: en/modules/admin/pages/initiatives.adoc

Some real examples from the city of Barcelona are:

* To add an item to the Municipal Council or Districts agendas
* To perform a participatory process
* To create a new assembly
* To approve or modify a regulation
* To request the holding of a citizen consultation
* To call for assemblies (neighborhood councils or public audiences)

=== New initiative type form

image:admin_initiative_type.png[New initiative type form]

.Admin new initiative type form: General information
|===
|Field |Type |Description

|Title
|Required
|Title of the initiative type. For instance, "New park at city center".

|Description
|Required
|Explanation of what can be done with this initiative type.

|Banner image
|Required
|An image for this initiative type. NOTE: At the moment this is not used.

|===

.Admin new initiative type form: Options
|===
|Field |Type |Description

|Signature type
|Optional
|Which kind of signature type will be allowed. Options are: Online, In-person and Mixed. The Mixed option allows a participant to choose which signature type will have the initiative.

|Enable attachments
|Optional

> |Wheter it'll allow to upload attachments to initiatives.

Reviewer: ```suggestion
|Whether it'll allow to upload attachments to initiatives.
```

|Enable participants to undo their online signatures
|Optional
|Can a participant undo the signature once it have already signed?

|Enable authors to choose the end of the signature collection period
|Optional
|Can the authors of an initiative change the end of the signature collection period?

|Enable authors to choose the area for their initiative
|Optional
|Can the authors choose the area of their initiative?

|Enable child scope signatures
|Optional
|This config flag doesn't support offline votes, it enables sub-scopes and works with an authorization handler that associates a scope to the user, make sure you select that authorization, bellow in authorization settings. For it to work, scopes need to be configured in a hierarchical way : 1 Parent - N Child.

|Only allow global scope initiatives creation
|Optional
|Tick this flag if you enabled "Child scope signature" and configured the global scope as your parent scope. By enabling this, initiative type selection will be skipped in the initiative creation wizard.

|Enable promoting committee
|Optional

> |Wheter if it will allow individual initiatives (made by only one person) or it will always have to have a collective (multiple promoters, being multiple individuals or a user group).

Reviewer: ```suggestion
|Whether it will only allow individual initiatives (made by only one person) or it will always have to have a collective (multiple promoters, being multiple individuals or a user group).
```

== 2. Send initiative to technical validation

This action is done by **the author**.

After it has been reviewed by the author, it can be sent to "Technical validation", but it depends in the configuration of the initiative type, in whether the options "Enable promoting committee" and "Minimum of committee members" are configured.

=== Promotion committee

An initiative type can optionally be supported by a promotion committee, with a minimum
number of committee members. Once the user has created the initiative and before it can be
sent for technical validation they need to invite committee members to promote it.

When the user has created the initiative they will be given a link to share with possible
committee members, which will look something like `/initiatives/.../committee_requests/new`

When a prospective committee member opens the link, they can click a button which allows
them to request to be part of the committee. The initiative author then needs to approve
each request, by opening the Admin Dashboard link in the user menu, editing their
initiative, clicking "Committee members" and then approving each member.

> image:initiative_commitee_members_send_to_technical_false.png[Initiative promotor committee: edit screen]

Reviewer: ```suggestion
image:initiative_commitee_members_send_to_technical_false.png[Initiative promotor committee: edit screen]

The initiative cannot be sent to technical validation until it meets the minimum number of committee members required. You can find the invitation link in the edit form.

```

> image:initiative_edit_commitee_members.png[Initiative promotor committee: edit screen]

Reviewer: ```suggestion
image:initiative_edit_commitee_members.png[Initiative promotor committee: edit screen]

Share the invitation link with the people you'd like to have on your committee. 

```

> image:initiative_commitee_members_sent.png[Initiative promotor committee: edit screen]

Reviewer: ```suggestion
image:initiative_commitee_members_ask.png[Initiative promotor committee: edit screen]

The people invited with the link will be able to read the initiative and ask to be part of the committee.

```

> image:initiative_commitee_members_ask.png[Initiative promotor committee: edit screen]

Reviewer: ```suggestion
image:initiative_commitee_members_sent.png[Initiative promotor committee: edit screen]

The request is sent and the author can then and accept or refuse it in the initiative edit form. 

```

> image:initiative_commitee_members_approve.png[Initiative promotor committee: edit screen]

Reviewer: ```suggestion
image:initiative_commitee_members_approve.png[Initiative promotor committee: edit screen]

```
